### PR TITLE
update changelog 

### PR DESCRIPTION
**Pull request by backwind1233**
- update changelog for  3.5.0-beta.1 entry

### CODE REVIEW

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,30 @@
...
+    ```java
+        @GetMapping("group1Id")
+        @ResponseBody
+        @PreAuthorize("hasRole('ROLE_<group1-id>')")
+        public String group1Id() {
+            return "group1Id message";
+        }
 
+    ```
+
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:30]: How about we change to a table describing the properties we added in this feature?

**Review by saragluna — DISMISSED** [review]

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md (hunk 1 above)
Comment (backwind1233) [reply sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md]: good idea

**Comment by chenrujun** [comment]
Hi, @backwind1233 .
Please resolve the conflicts.

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md [hunk 2]
```diff
@@ -5,11 +5,26 @@
...
+  ```
+  | Parameter           | Description                                             |
+  | ------------------- | ------------------------------------------------------------ |
+  | allowed-group-names | if `enable-full-list` is `false`, create `GrantedAuthority` with groupNames which belong to user and `allowed-group-names` |
+  | allowed-group-ids   | if `enable-full-list` is `false`, create `GrantedAuthority` with groupIds which belong to user and `allowed-group-ids` |
+  | enable-full-list    | default is `false`.<br> if the value is `true`, create `GrantedAuthority` only with user's  all groupIds, ignore group names|
 
-
+### Key Bug Fixes
+- Fix the issue [#21036](https://github.com/Azure/azure-sdk-for-java/issues/21036) where the AAD B2C starter cannot fetch the OpenID Connect metadata document via issuer.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:26]:
```
- Fix bug that AAD B2C starter cannot fetch the OpenID Connect metadata document via issuer. ([#21036](https://github.com/Azure/azure-sdk-for-java/issues/21036) ).
```

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md [hunk 3]
```diff
@@ -5,11 +5,26 @@
 
 ## 3.5.0 (2021-05-24)
 ### New Features
+- Add `AADB2CTrustedIssuerRepository` to manage the trusted issuer in AAD B2C.
 - Upgrade to [spring-boot-dependencies:2.4.5](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.4.5/spring-boot-dependencies-2.4.5.pom).
 - Upgrade to [spring-cloud-dependencies:2020.0.2](https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2020.0.2/spring-cloud-dependencies-2020.0.2.pom).
 - Enable property azure.activedirectory.redirect-uri-template.([#21116](https://github.com/Azure/azure-sdk-for-java/issues/21116))
+- Support creating `GrantedAuthority` by groupId and groupName for web application.([#20218])
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:12]: nit: Could you please keep the same style with line 11?

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md [hunk 4]
```diff
@@ -5,11 +5,26 @@
...
+  user-group:
+      allowed-group-names: group1,group2
+      allowed-group-ids: <group1-id>,<group2-id>
+      enable-full-list: false  
+  ```
+  | Parameter           | Description                                             |
+  | ------------------- | ------------------------------------------------------------ |
+  | allowed-group-names | if `enable-full-list` is `false`, create `GrantedAuthority` with groupNames which belong to user and `allowed-group-names` |
+  | allowed-group-ids   | if `enable-full-list` is `false`, create `GrantedAuthority` with groupIds which belong to user and `allowed-group-ids` |
+  | enable-full-list    | default is `false`.<br> if the value is `true`, create `GrantedAuthority` only with user's  all groupIds, ignore group names|
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:23]: Why ignore group names? If the logic is ignore group names, `allow-all-group-ids-only` is a better property name.

File: sdk/spring/azure-spring-boot-starter/CHANGELOG.md [hunk 5]
```diff
@@ -5,10 +5,26 @@
 
 ## 3.5.0 (2021-05-24)
 ### New Features
+- Add `AADB2CTrustedIssuerRepository` to manage the trusted issuer in AAD B2C.
 - Upgrade to [spring-boot-dependencies:2.4.5](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.4.5/spring-boot-dependencies-2.4.5.pom).
 - Upgrade to [spring-cloud-dependencies:2020.0.2](https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2020.0.2/spring-cloud-dependencies-2020.0.2.pom).
+- Enable property azure.activedirectory.redirect-uri-template.([#21116](https://github.com/Azure/azure-sdk-for-java/issues/21116))
+- Support creating `GrantedAuthority` by groupId and groupName for web application.([#20218])
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter/CHANGELOG.md:12]: Same here.

File: sdk/spring/azure-spring-boot-starter/CHANGELOG.md [hunk 6]
```diff
@@ -5,10 +5,26 @@
...
+  ```
+  | Parameter           | Description                                             |
+  | ------------------- | ------------------------------------------------------------ |
+  | allowed-group-names | if `enable-full-list` is `false`, create `GrantedAuthority` with groupNames which belong to user and `allowed-group-names` |
+  | allowed-group-ids   | if `enable-full-list` is `false`, create `GrantedAuthority` with groupIds which belong to user and `allowed-group-ids` |
+  | enable-full-list    | default is `false`.<br> if the value is `true`, create `GrantedAuthority` only with user's  all groupIds, ignore group names|
 
-
+### Key Bug Fixes
+- Fix the issue [#21036](https://github.com/Azure/azure-sdk-for-java/issues/21036) where the AAD B2C starter cannot fetch the OpenID Connect metadata document via issuer.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter/CHANGELOG.md:26]: Here.

File: sdk/spring/azure-spring-boot/CHANGELOG.md [hunk 7]
```diff
@@ -6,7 +6,21 @@
 ## 3.5.0 (2021-05-24)
 ### New Features
 - Add `AADB2CTrustedIssuerRepository` to manage the trusted issuer in AAD B2C.
-- Enable property azure.activedirectory.redirect-uri-template. ([#21116](https://github.com/Azure/azure-sdk-for-java/issues/21116))
+- Upgrade to [spring-boot-dependencies:2.4.5](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.4.5/spring-boot-dependencies-2.4.5.pom).
+- Upgrade to [spring-cloud-dependencies:2020.0.2](https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2020.0.2/spring-cloud-dependencies-2020.0.2.pom).
+- Enable property azure.activedirectory.redirect-uri-template.([#21116](https://github.com/Azure/azure-sdk-for-java/issues/21116))
+- Support creating `GrantedAuthority` by groupId and groupName for web application.([#20218])
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot/CHANGELOG.md:12]: Here.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md (hunk 4 above)
Comment (backwind1233) [reply sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:23]: yes, I agree

Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:23]: How about:
1. Delete `enable-full-list`? 
2. Use `allowed-group-ids = *` to implement function of `enable-full-list`.
3. Let `allowed-group-ids = *` can work together with `allowed-group-names`.

Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:23]: As we discussed here: https://github.com/Azure/azure-sdk-for-java/issues/20218#issuecomment-827476074

**Review by chenrujun — CHANGES_REQUESTED** [review]

**Comment by backwind1233** [comment]
https://github.com/Azure/azure-sdk-for-java/issues/21983

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**